domain/cloud/bootstrap/bootstrap.go

```diff
@@ -20,7 +21,7 @@ import (
 // InsertCloud inserts the initial cloud during bootstrap.
 func InsertCloud(cloud cloud.Cloud) func(context.Context, database.TxnRunner) error {
 	return func(ctx context.Context, db database.TxnRunner) error {
-		return errors.Trace(db.StdTxn(ctx, func(ctx context.Context, tx *sql.Tx) error {
+		return errors.Trace(db.Txn(ctx, func(ctx context.Context, tx *sqlair.TX) error {
 			cloudUUID, err := uuid.NewUUID()
```

Comment:
I know you didn't do this, but whilst you're here, can you move the `cloudUUID, err := uuid.NewUUID()` line outside of the `Txn` method.

Reply:
Changed